migrations/versions/test_7c929f8e-bd11-4283-9603-40613839d23a.py

```diff
@@ -0,0 +1,5 @@
+# Migration script generated for test_migration_2
```

Comment:
this should live somewhere in the tests directory

Reply:
Will delete it and use mock instead